<?xml version="1.0"?>
<launch>
  <arg name="mav_name" default="firefly"/>
  <arg name="world_name" default="basic"/>
  <arg name="enable_logging" default="false"/>
  <arg name="image_def" default="vi_sensor/camera_depth/depth"/>

  <group ns="$(arg mav_name)">
      <param name="uav_name" value="firefly"/>
      <param name="uav_velocity" value="3.0"/>
      <param name="initial_hovering_height" value="3.0"/>
      <node pkg="geometric_controller" name="PositionVelocityController" type="position_velocity_controller.py" respawn="true"/>
      <node pkg="geometric_controller" name="navfunc3d_nobstacle" type="3DNF_SW_nObstacle_R1_ROSnode.py" respawn="true" output="screen"/>
      <remap from="odometry" to="odometry_sensor1/odometry" />	
     
  </group>

</launch>
